Libraries/StyleSheet/LayoutPropTypes.js

```diff
@@ -104,10 +325,27 @@ var LayoutPropTypes = {
     'stretch'
   ]),
 
-  // https://developer.mozilla.org/en-US/docs/Web/CSS/flex
+  /** In React Native `flex` does not work the same way that it does in CSS.
+   *  parameter. `flex` is a number rather than a string, and it works
+   *  according to the `css-layout` library
+   *  at https://github.com/facebook/css-layout . Typically, you set `flex`
+   *  to be a positive number that controls how much of the available space
+   *  this component should take up.
+   */
```

Comment:
It would be nice to say a little bit more about flex but not sure what. Perhaps more about what the positive number means.